code/game/area/Space Station 13 areas.dm

```diff
@@ -197,7 +197,7 @@ var/global/list/adminbusteleportlocs = list()
 
 /area/shuttle
 	requires_power = 0
-	dynamic_lighting = 1 //Lighting STILL disabled, even with the new bay engine, because lighting doesn't play nice with our shuttles, might just be our shuttle code, or the small changes in the lighting engine we have from bay.
+	dynamic_lighting = 0
```

Comment:
what's the deal with shuttles losing dynamic lighting?

Reply:
Calling it "dynamic lighting" is a misnommer now, since with this var you just get the turf to always be lit, similar to space tiles. We don't need on shuttles anymore, since they have lights. As the comment said, "dynamic_lighting = 1" just means the lighting is disabled.

IIRC it works fine now.